packages/react-dom-bindings/src/client/ReactDOMHostConfig.js

```diff
@@ -902,16 +910,25 @@ function getNextHydratable(node) {
       if (nodeType === ELEMENT_NODE) {
         const element: Element = (node: any);
         switch (element.tagName) {
+          case 'TITLE':
+          case 'META':
+          case 'BASE':
+          case 'HTML':
+          case 'HEAD':
+          case 'BODY': {
+            continue;
+          }
           case 'LINK': {
             const linkEl: HTMLLinkElement = (element: any);
-            const rel = linkEl.rel;
+            // All links that are server rendered are resources except
+            // stylesheets that do not have a precedence
             if (
-              rel === 'preload' ||
-              (rel === 'stylesheet' && linkEl.hasAttribute('data-precedence'))
+              linkEl.rel === 'stylesheet' &&
+              !linkEl.hasAttribute('data-precedence')
             ) {
-              continue;
+              break;
             }
-            break;
+            continue;
           }
```

Comment:
This is confusing but before it would only skip a link if it was a preload or stylesheet with precedence and now it skips all links unless it is a stylesheet without precedence.